### PR TITLE
feat(collapsible): add new component collapsible

### PR DESCRIPTION
**Pull request by YeSuX**

### CODE REVIEW

File: packages/components/collapsible/src/collapsible.ts [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      defaultProp: defaultOpen.value,
+    })
+
+    collapsibleProvider({
+      contentId: useId(),
+      disabled: disable,
+      open: state,
+      onOpenToggle() {
+        // TODO: Need Help!!!
+        state.value = !state.value
```
Comment (YeSuX) [inline packages/components/collapsible/src/collapsible.ts:75]: Need Help !!!

Comment (YeSuX) [reply packages/components/collapsible/src/collapsible.ts:75]: I spent a long time trying to solve the issue of modifying data between parent and child components, but still haven't found a solution.

File: packages/components/avatar/src/stories/avatar.stories.ts
Comment (productdevbook) [inline packages/components/avatar/src/stories/avatar.stories.ts:1]: dont delete file

File: packages/components/collapsible/src/collapsible.ts (hunk 1 above)
Comment (productdevbook) [reply packages/components/collapsible/src/collapsible.ts]: I've fixed many builds and bugs. [check new commit.](https://github.com/oku-ui/primitives/pull/199/commits/838d891b073c63adbff1d625674080219018096a)

Comment (YeSuX) [reply packages/components/collapsible/src/collapsible.ts]: Thank you! You're amazing. I'll learn from you.

File: packages/components/collapsible/src/collapsible.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+      >,
+      default: undefined,
+    },
+    defaultOpen: {
+      type: Boolean,
+    },
+    open: {
+      type: Boolean as PropType<boolean | undefined>,
+    },
+    disable: {
```
Comment (dammy001) [inline packages/components/collapsible/src/collapsible.ts:43]: Shouldn't this be `disabled`?

**Comment by productdevbook** [comment]
@YeSuX 

image


https://601857eb614bae0021c9b9dd-jvztzayxuq.chromatic.com/?path=/story/components-collapsible--chromatic

you need to add all fields here for storybook

**Comment by YeSuX** [comment]
> 
I have filled in all the fields except for animation, but for some of them, I wrote them together.

**Review by productdevbook — APPROVED** [review]

**Comment by productdevbook** [comment]
:heart: thank you i like it

**Review by dammy001 — APPROVED** [review]
LGTM

**Pull request merged**